docs/app/Examples/modules/Popup/Variations/PopupFluidExample.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Message, Button, Popup, Grid } from 'semantic-ui-react'
+
+export default function PopupFluidExample() {
+  return (
+    <div>
+      <Message
+        warning
+        header='Work in Progress'
+        content='This one needs to be resized according to the parrent offset element.'
+      />
```

Comment:
Suggest pulling this example and allowing a future PR to add it since it needs more work.  Note, when adding Messages, add them to the `index.js` file so that it doesn't show up in the example code editor in the docs:

**docs/app/Examples/modules/Popup/Variations/index.js**
```jsx
<ComponentExample
  title='Fluid'
  description='A fluid popup will take up the entire width of its offset container'
  examplePath='modules/Popup/Variations/PopupFluidExample'
>
  <Message
    warning
    header='Work in Progress'
    content='This one needs to be resized according to the parrent offset element.'
  />
</ComponentExample>
```